dune

```diff
@@ -192,6 +194,7 @@
   interf
   interval
   linear
+  internal_assembler
```

Comment:
Add new files to compiler-libs (there is a helper script in `scripts` subdirectory).